sinch/domains/verification/endpoints/get_verification_by_identity.py

```diff
@@ -0,0 +1,27 @@
+from sinch.core.models.http_response import HTTPResponse
+from sinch.domains.verification.endpoints.verification_endpoint import VerificationEndpoint
+from sinch.core.enums import HTTPAuthentication, HTTPMethods
+from sinch.domains.verification.models.requests import GetVerificationByIdentityRequest
+from sinch.domains.verification.models.responses import GetVerificationByIdentityResponse
+
+
+class GetVerificationByIdentityEndpoint(VerificationEndpoint):
+    ENDPOINT_URL = "{origin}/verification/v1/verifications/{method}/number/{endpoint}"
```

Comment:
Naming is confusing 
- `GetVerificationByIdentityEndpoint` is for `Verification Status` ? (https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verification-status/)
- `ReportVerificationUsingIdentityEndpoint` is for `Verifications` from docs (https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verifications/)

Why not :
- having `Status` for Status APIs ?
- common pattern for function name and file names : `By` vs `Using` ?

Reply:
I've made naming more pleasant :) 
fixed